docs/queries/queries.md

```diff
@@ -363,7 +393,7 @@ user = await User.query.update(email="bar@foo.com")
 
 ### Get
 
-Obtains a single record from the database.
+Obtains a single record from the database. When using keywords (or no args) the cache is used.
```

Comment:
Same thing. No need the extra that was added. Too much info for unnecessary reasons